convert2rhel/actions/__init__.py

```diff
@@ -296,41 +345,41 @@ class ActionMessage(ActionMessageBase):
     A class that defines the contents and rules for messages set through :meth:`Action.add_message`.
     """
 
-    def __init__(self, level=None, id=None, message=None):
-        if not (id and level and message):
-            raise InvalidMessageError("Messages require id, level and message fields")
+    def __init__(self, level=None, id=None, title=None, description=None, diagnosis=None, remediation=None):
+        if not (id and level and title and description):
+            raise InvalidMessageError("Messages require id, level, title and description fields")
 
         # None of the result status codes are legal as a message.  So we error if any
         # of them were given here.
         if not (STATUS_CODE["SUCCESS"] < STATUS_CODE[level] < STATUS_CODE["SKIP"]):
             raise InvalidMessageError("Invalid level '%s', set for a non-result message" % level)
 
-        super(ActionMessage, self).__init__(level, id, message)
+        super(ActionMessage, self).__init__(level, id, title, description, diagnosis, remediation)
 
 
 class ActionResult(ActionMessageBase):
     """
     A class that defines content and rules for messages set through :meth:`Action.set_result`.
     """
 
-    def __init__(self, level="SUCCESS", id=None, message=""):
+    def __init__(self, level="SUCCESS", id=None, title="", description="", diagnosis="", remediation=""):
 
         if STATUS_CODE[level] >= STATUS_CODE["SKIP"]:
-            if not id and not message:
-                raise InvalidMessageError("Non-success results require an id and a message")
+            if not id and not title:
+                raise InvalidMessageError("Non-success results require an id, title and description")
```

Comment:
```suggestion
                raise InvalidMessageError("Non-success results require an id, title")
```

There is no description in the conditional.